### PR TITLE
Add ITDev logo as sponsor

### LINKED ISSUE #364: Add ITDev logo to website

### PR DESCRIPTION
**Pull request by antoinepetty**
Closes #364

### CODE REVIEW

**Comment by antoinepetty** [comment]
Preview
image

**Review by sedders123 — APPROVED** [review]
No SVG :(

**Comment by antoinepetty** [comment]
Unfortunately they don't have a vector version of their logo. Will wait for final confirmation before merging this

**Pull request merged**
